feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -24,7 +25,12 @@ class IdentityManager(
 
     @Synchronized
     fun configure(appUserID: String?) {
+        if (appUserID?.isEmpty() == true) {
+            log(LogIntent.WARNING, IdentityStrings.EMPTY_APP_USER_ID_WILL_BECOME_ANONYMOUS)
+        }
+
         val appUserIDToUse = appUserID
+            ?.notEmpty
```

Comment:
Instead of doing the extension, this can be written as "appUserID?.takeUnless { it.isBlank() }", which it feels more Kotlin to me. 

I had to see what `notEmpty` was doing to understand what it meant. `empty` in Kotlin is typically used for `""`. Something is `blank` when it's either the `empty` string or if it consist solely of whitespace character. And `notEmpty` is actually checking if it's not `blank`.

Maybe calling the function `nullIfBlank()`. Also if you look into the standard library extension functions for string, they are normally extension functions and not extension properties.

Reply:
That's exactly the type of feedback I was looking for, thanks! I'll use `appUserID?.takeUnless { it.isBlank() }`